src/main/kotlin/com/ampnet/reportservice/controller/AdminController.kt

```diff
@@ -26,11 +31,24 @@ class AdminController(
         @RequestParam(name = "to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) to: LocalDate?
     ): ResponseEntity<ByteArray> {
         val user = ControllerUtils.getUserPrincipalFromSecurityContext()
-        logger.debug {
+        logger.info {
             "Received request to get users accounts summary for all the active users"
         }
         val periodRequest = PeriodServiceRequest(from, to)
         val pdfContents = reportingService.generatePdfReportForAllActiveUsers(user, periodRequest)
         return ResponseEntity(pdfContents, ControllerUtils.getHttpHeadersForPdf(), HttpStatus.OK)
     }
+
+    @GetMapping("/admin/report/xlsx")
+    @PreAuthorize("hasAuthority(T(com.ampnet.reportservice.enums.PrivilegeType).PRA_PROFILE)")
+    fun getXlsxReport(@RequestParam type: XlsxType): ResponseEntity<ByteArray> {
+        val user = ControllerUtils.getUserPrincipalFromSecurityContext()
```

Comment:
Request param should be in lowercase. You can put a mapper to map input values into your XlsxType enum which would ignore case.